templates/terraform/resource.erb

```diff
@@ -193,6 +193,10 @@ func resource<%= resource_name -%>Create(d *schema.ResourceData, meta interface{
 <%  end -%>
     res, err := sendRequestWithTimeout(config, "<%= object.create_verb.to_s.upcase -%>", <% if has_project || object.supports_indirect_user_project_override %>project<% else %>""<% end %>, url, obj, d.Timeout(schema.TimeoutCreate) <%= object.error_retry_predicates ? ", " + object.error_retry_predicates.join(',') : "" -%>)
     if err != nil {
+        <%  # Async block adds post-create failure code after operation instead.
```

Comment:
nit: erb-only lines in this file are generally left-aligned, with their ruby contents indented